libusbcommunication/src/c/usb.c

```diff
@@ -0,0 +1,61 @@
+#include <jni.h>
+#include <libusb.h>
+#include <stdio.h>
+
+#include "log.h"
+
+#define TAG "native_libusbcom"
+
+static void print_devs(libusb_device **devs)
+{
+    libusb_device *dev;
+    int i = 0, j = 0;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable 'j' is assigned a value that is never used.](https://app.codacy.com/manual/magnusja/libaums/pullRequest?prid=4798940)